scarb-metadata/tests/command.rs

```diff
@@ -69,7 +69,7 @@ fn manifest_path() {
 
 fn init_project(t: &TempDir) {
     Command::new(scarb_bin())
-        .args(["init", "--name", "hello"])
+        .args(["init", "--name", "hello", "--test-runner", "cairo-test"])
```

Comment:
We have a funny test case here: https://github.com/software-mansion/scarb/blob/a142df1dd521b8a78cd265877d1cd753a7aa437e/.github/workflows/ci.yml#L67-L78
Which runs tests from this package with older version of Scarb, to ensure it's still compatible in regards to metadata format. 
This older version of scarb will not accept `test-runner` as an argument.

Reply:
But as far as I understand, the normal tests won't pass. They need `--test-runner` as they also run this package.

Reply:
Yeah, but this test is very important to me (it can warn us about unintentional breaking changes in the metadata format), so we need to think about some solution that works for both cases 😄 

You can always run `scarb init --help` and parse the output to see if it accepts the test runner flag or not. 

Another approach you can consider is reading the value from env - https://docs.rs/clap/latest/clap/struct.Arg.html#method.env
Say, `SCARB_INIT_TEST_RUNNER` or similar. It might be also useful in `snforge init` which calls Scarb under the hood.

Reply:
I think the env approach is better, but I won't be able to export it in CI with my permissions. So when I apply that fix, could you do that for me?

Reply:
Just add it here with `.env` on Command.